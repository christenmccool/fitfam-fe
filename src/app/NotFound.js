import React from 'react';

import Container from '@mui/material/Container';
import Box from '@mui/material/Box';
import Typography from '@mui/material/Typography';

/** Not Found page
 */
const NotFound = () => {

  return (
    <Container maxWidth="sm" align="center" sx={{backgroundColor: "#FFF", borderRadius: '10px'}}>
      <Box my={4} p={5}>
          <Typography variant="h6">
            The page you are looking for cannot be found
          </Typography>
      </Box>
    </Container>
  )
}

export default NotFound;