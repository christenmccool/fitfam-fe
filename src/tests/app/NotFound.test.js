import React from "react";
import { render, screen } from "@testing-library/react";
import { MemoryRouter } from "react-router-dom";

import NotFound from "../../app/NotFound";


it('renders without crashing', () => {
  render(
    <MemoryRouter>
      <NotFound />
    </MemoryRouter>
  );
});


it("matches snapshot", function () {
  const { asFragment } = render(
    <MemoryRouter>
      <NotFound />
    </MemoryRouter>
  );
  expect(asFragment()).toMatchSnapshot();
});


it("renders text", function () {
  render(
    <MemoryRouter>
      <NotFound />
    </MemoryRouter>
  );
  expect(screen.getByText("The page you are looking for cannot be found")).toBeInTheDocument();
});



