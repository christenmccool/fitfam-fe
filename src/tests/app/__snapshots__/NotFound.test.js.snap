// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    align="center"
    class="MuiContainer-root MuiContainer-maxWidthSm css-ead4d9-MuiContainer-root"
  >
    <div
      class="MuiBox-root css-19rm58x"
    >
      <h6
        class="MuiTypography-root MuiTypography-h6 css-2ulfj5-MuiTypography-root"
      >
        The page you are looking for cannot be found
      </h6>
    </div>
  </div>
</DocumentFragment>
`;
